docs/Style.thy

```diff
@@ -87,11 +88,18 @@ text \<open>
   the closing bracket on the same line as the ending text to not waste too much vertical space.
   Indent text by 2 inside the @{command text} area. This achieves visual separation.\<close>
 
+
 section \<open>Indentation\<close>
 
 text \<open>
-  Isabelle code is much easier to maintain when indented consistently. In apply style proofs we
-  indent by 2 spaces, and add an additional space for every additional subgoal.
+  Isabelle code is much easier to maintain when indented consistently.
+  When in doubt, and not constrained by vertically aligning items or subgoal-count offsets, use
```

Comment:
Extremely minor grammar nitpick
```suggestion
  When in doubt and not constrained by vertically aligning items or subgoal-count offsets, use
```

Reply:
*shrug* ... if it helps, I'll add the comma

Reply:
I'm suggesting to remove the comma, I don't think it's right.

Reply:
Oh, yes, will do then